src/components/app/DocumentList/index.js

```diff
@@ -0,0 +1,855 @@
+    });
+  };
+
+  /**
+   * @method updateQuickActions
+   * @summary Trigger the QuickActions component to fetch quick actions for the new selection
+   */
+  updateQuickActions = (childSelection) => {
+    if (this.quickActionsComponent) {
+      this.quickActionsComponent.updateActions(childSelection);
+    }
+  };
+
+  /**
+   * @method loadSupportAttributeFlag
+   * @summary Load supportAttribute of the selected row from the table.
+   */
+  loadSupportAttributeFlag = ({ selected }) => {
+    const {
+      reduxData: { rowData },
+    } = this.props;
+
+    if (!rowData) {
+      return;
+    }
+    const rows = getRowsData(rowData.get('1'));
+
+    if (selected.length === 1) {
+      const selectedRow = rows.find((row) => row.id === selected[0]);
+
+      this.setState({
+        supportAttribute: selectedRow && selectedRow.supportAttributes,
+      });
+    } else {
+      this.setState({
+        supportAttribute: false,
+      });
+    }
+  };
+
+  // TODO: I think this should be stored in redux too
+  /**
+   * @method clearStaticFilters
+   * @summary ToDo: Describe the method.
+   */
+  clearStaticFilters = (filterId) => {
+    const { push, windowType, viewId } = this.props;
+
+    deleteStaticFilter(windowType, viewId, filterId).then((response) => {
+      this.setState({ staticFilterCleared: true }, () =>
+        push(`/window/${windowType}?viewId=${response.data.viewId}`)
+      );
+    });
+  };
+
+  // FETCHING LAYOUT && DATA -------------------------------------------------
+  /**
+   * @method fetchLayoutAndData
+   * @summary ToDo: Describe the method.
+   */
+  fetchLayoutAndData = (isNewFilter, locationAreaSearch) => {
+    const {
+      windowType,
+      type,
+      viewProfileId,
+      setModalTitle,
+      viewId,
+      fetchLayout,
+      updateRawModal,
+    } = this.props;
+
+    this.setState({ triggerSpinner: true });
+
+    fetchLayout(windowType, type, viewProfileId)
+      .then((response) => {
+        if (this.mounted) {
+          const { allowedCloseActions } = response;
+
+          if (allowedCloseActions) {
+            updateRawModal(windowType, { allowedCloseActions });
+          }
+
+          if (viewId) {
+            this.connectWebSocket(viewId);
+
+            if (!isNewFilter) {
+              this.browseView();
+            } else {
+              this.filterView(locationAreaSearch);
+            }
+          } else {
+            this.createView();
+          }
+
+          // TODO: Should we handle this in the action creator ?
+          setModalTitle && setModalTitle(response.data.caption);
+        }
+      })
+      .catch(() => {
+        this.setState({ triggerSpinner: false });
+      });
+  };
+
+  /**
+   * @method browseView
+   * @summary If viewId exists, than browse that view.
+   */
+  browseView = () => {
+    const { viewId, page, sort } = this.props;
+    const { filtersActive } = this.state;
+    const locationSearchFilter = filtersActive.has(`location-area-search`);
```

Comment:
Use single quotes for string pls

Reply:
I suggest we switch to using backtick, as this allows us to use string literals everywhere. It's ES6, and we're in 2020 already.

Reply:
https://eslint.org/docs/rules/quotes . We have set the single already and backticks are allowed but in this case there is no ${var} within so for consistency can be set to single quotes and we improve the code.
<img width="739" alt="Screenshot 2020-03-03 at 19 40 36" src="https://user-images.githubusercontent.com/1708561/75803405-04293900-5d87-11ea-814e-33275dd696fd.png">

Reply:
I don't agree this improves the code in any way. Using backticks you have only one way to indicate strings no matter if it's a literal or not.

Reply:
switching in codacy to use that for all code would mean modifying the strings all over to use only backticks. Anyways I can agree as it is even so it would improve the mark on codacy to use just simple quotes in here.